### ISSUE TITLE
The `v update` command does not update the ui module

### ISSUE THREAD

**Issue by sereg1k** [vlang/v#5146]
**V version:** V 0.1.27 [f87e872](https://github.com/vlang/v/commit/f87e872fa27962b3911feec36e63d1623858045c)
**OS:** Linux

**What did you do?**
I installed several any modules:
```
v install author.module_name1
v install ui
```
Then
`v update`

**What did you expect to see?**
```
Updating module "author.module_name1"...
Updating module "ui"...
```
 
**What did you see instead?**
```
Updating module "author.module_name1"...
```
**PS:**
If you use `v update ui` for this, the update is successful.

**Comment by spytheman** [comment]
Hm, for me it works:
![v install ui](https://url4e.com/gyazo/images/c97ca85b.png)

@sereg1k can you try:
a) `( cd ~/.vmodules/ui/ ; git status )`
b) `rm -rf ~/.vmodules/ui` 
c) `v install ui` several times in a row?
It may be that you have locally changed files there, which prevents `git pull` from working.

**Comment by spytheman** [comment]
Hm, scratch that ... I misunderstood your issue.
You mean that `v update` skips the `ui` - that is a problem indeed. I will fix it soon.

**Comment by spytheman** [comment]
Should be fixed in d7bb887c . @sereg1k please do `v up && v update` to check.

**Comment by sereg1k** [comment]
Yes, there is no problem with the [d7bb887](https://github.com/vlang/v/commit/d7bb887c2a8b940cee2bbc38612ea0d4c34ec5b7) commit.
